app/assets/javascripts/revised/pages/info/manufacturers.js.coffee

```diff
@@ -1,13 +1,11 @@
-class BikeIndex.Views.ContentManufacturers extends Backbone.View
-
-  initialize: ->
+class BikeIndex.InfoManufacturers extends BikeIndex
```

Comment:
We aren't using Backbone in the revised view - so the classes are instantiated differently. If you look at one of the other files in this folder (e.g. [where.coffee](https://github.com/bikeindex/bike_index/blob/master/app/assets/javascripts/revised/pages/info/where.coffee), you can see the way we name classes.